web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1354,7 +1354,17 @@ fixmystreet.display = {
             }
 
             var found = html.match(/<title>([\s\S]*?)<\/title>/);
-            var page_title = found[1];
+            var map = {
```

Comment:
I feel like maybe this could do with a comment, explaining:

1. We’re unencoding HTML entities in the title string, to make it suitable for use with`document.title` later
2. We’re only worried about these 5 particular entities because they’re the only 5 that the `html_filter` method of `FixMyStreet/Template.pm` encodes in the first place